cmd/porter/credentials.go

```diff
@@ -0,0 +1,151 @@
+package main
+
+import (
+	"github.com/deislabs/porter/pkg/porter"
+	"github.com/deislabs/porter/pkg/printer"
+	"github.com/spf13/cobra"
+)
+
```

Comment:
Do we want a `creds` alias?

Reply:
Yeah seems like it would be nice to have both aliased because we all suck at typing.